### PR TITLE
fix(src/test): Address lint issues + setup lint/format CI action

### PR DESCRIPTION
**Pull request by igoroctaviano**
- Address lint issues
- Add format and ci lint action to avoid merging broken/unformatted code

### CODE REVIEW

File: src/sr/templates.js [hunk 1]
```diff
@@ -302,7 +292,7 @@ class MeasurementStatisticalProperties extends Template {
                 value: options.authority,
                 relationshipType: RelationshipTypes.HAS_PROPERTIES
             });
-            this.push(authorityItem);
+            this.push(descriptionItem);
```
Comment (pieper) [inline src/sr/templates.js:295]: Looks like this was a bug before and isn't part of the code formatting?

**Review by pieper — COMMENTED** [review]
Looks great overall.  Just not sure about that one line.

File: src/sr/templates.js (hunk 1 above)
Comment (igoroctaviano) [reply src/sr/templates.js:295]: True but I catched it through lint

Comment (pieper) [reply src/sr/templates.js:295]: Cool!  Good catch 👍

**Pull request merged**